src/config.rs

```diff
@@ -1,27 +1,8 @@
-// Copyright (c) 2018 Pietro Albini <pietro@pietroalbini.org>
```

Comment:
These copyright notices seemed redundant as there is also a `LICENSE` file, if we do really want them I can add them back